### PR TITLE
feat(org-config): support supplying multiple org units

### PR DESCRIPTION
**Pull request by ipcrm**
### Summary

Enable supplying multiple target OU for org-config

### Issue

https://lacework.atlassian.net/browse/GROW-2545

### CODE REVIEW

**Review by afiune — APPROVED** [review]

File: cli/cmd/generate_aws.go [hunk 1]
```diff
@@ -498,10 +498,10 @@ func initGenerateAwsTfCommandFlags() {
 		"config_organization_id",
 		"",
 		"specify AWS organization ID for Config organization integration")
-	generateAwsTfCommand.PersistentFlags().StringVar(
+	generateAwsTfCommand.PersistentFlags().StringSliceVar(
 		&GenerateAwsCommandState.ConfigOrgUnit,
 		"config_organization_unit",
```
Comment (PengyuanZhao) [inline cli/cmd/generate_aws.go:503]: As it accepts multiple OUs, shall we rename it to `config_organization_units`?

File: cli/cmd/generate_aws.go [hunk 2]
```diff
@@ -54,7 +54,7 @@ var (
 	QuestionConfigOrgLWAccessKeyId    = "Lacework access key ID:"
 	QuestionConfigOrgLWSecretKey      = "Lacework secret key:"
 	QuestionConfigOrgId               = "AWS organization ID:"
-	QuestionConfigOrgUnit             = "AWS organization unit:"
+	QuestionConfigOrgUnit             = "AWS organization unit (multiple can be supplied comma separated):"
```
Comment (PengyuanZhao) [inline cli/cmd/generate_aws.go:57]:
```suggestion
	QuestionConfigOrgUnits             = "AWS organization units (multiple can be supplied comma separated):"
```

File: cli/cmd/generate_aws.go (hunk 1 above)
Comment (PengyuanZhao) [inline cli/cmd/generate_aws.go:503]:
```suggestion
		"config_organization_units",
```

File: cli/cmd/generate_aws.go [hunk 3]
```diff
@@ -841,22 +841,28 @@ func promptConfigQuestions(config *aws.GenerateAwsTfConfigurationArgs) error {
...
 		}
 
+		var orgUnit string
+		if err := survey.AskOne(
+			&survey.Input{Message: QuestionConfigOrgUnit, Default: strings.Join(config.ConfigOrgUnit, ",")}, &orgUnit,
+			survey.WithValidator(survey.Required), survey.WithIcons(customPromptIconsFunc(IconConfig)),
+		); err != nil {
+			return err
+		}
+		config.ConfigOrgUnit = strings.Split(orgUnit, ",")
```
Comment (PengyuanZhao) [inline cli/cmd/generate_aws.go:855]:
```suggestion
		config.ConfigOrgUnits = strings.Split(orgUnitsInput, ",")
```

File: cli/cmd/generate_aws.go [hunk 4]
```diff
@@ -841,22 +841,28 @@ func promptConfigQuestions(config *aws.GenerateAwsTfConfigurationArgs) error {
...
-				Icon:     IconConfig,
-				Prompt:   &survey.Input{Message: QuestionConfigOrgCfResourcePrefix, Default: config.ConfigOrgCfResourcePrefix},
-				Response: &config.ConfigOrgCfResourcePrefix,
-				Required: true,
-			},
 		}); err != nil {
 			return err
 		}
 
+		var orgUnit string
```
Comment (PengyuanZhao) [inline cli/cmd/generate_aws.go:848]:
```suggestion
		var orgUnitsInput string
```

File: lwgenerate/aws/aws.go [hunk 5]
```diff
@@ -189,7 +189,7 @@ type GenerateAwsTfConfigurationArgs struct {
 	ConfigOrgId string
 
 	// Config organization unit
-	ConfigOrgUnit string
+	ConfigOrgUnit []string
```
Comment (PengyuanZhao) [inline lwgenerate/aws/aws.go:192]:
```suggestion
	ConfigOrgUnits []string
```

File: lwgenerate/aws/aws.go [hunk 6]
```diff
@@ -471,7 +471,7 @@ func WithConfigOrgId(orgId string) AwsTerraformModifier {
 }
 
 // WithConfigOrgUnit Set Config org unit
-func WithConfigOrgUnit(orgUnit string) AwsTerraformModifier {
+func WithConfigOrgUnit(orgUnit []string) AwsTerraformModifier {
```
Comment (PengyuanZhao) [inline lwgenerate/aws/aws.go:474]:
```suggestion
func WithConfigOrgUnits(orgUnits []string) AwsTerraformModifier {
```

**Pull request merged**